cross/gpac/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = $(SPK_NAME)
+PKG_VERS = $(SPK_VERS)
```

Comment:
cross/gpac is first and spk/gpac depends on it.
please avoid the use of SPK_ variables in cross/gpac/Makefile

general rule:
SPK_NAME must always be lower case, but PKG_NAME matches the downloaded file and might be different to SPK_NAME

Reply:
Changes applied accordingly.  Though I would avoid declaring the same values - e.g. version and revision - at two different locations.